libqtile/core/manager.py

```diff
@@ -104,11 +104,8 @@ def load_config(self):
             self.config.load()
             self.config.validate()
         except Exception as e:
-            logger.exception('Error while reading config file (%s)', e)
-            self.config = confreader.Config()
-            from libqtile.widget import TextBox
-            widgets = self.config.screens[0].bottom.widgets
-            widgets.insert(0, TextBox('Config Err!'))
+            msg = "Woops! Error in config: " + e.__class__.__name__
+            send_notification("Qtile Config", msg, timeout=20000)
```

Comment:
Let's make this consistent with the error in cmd_validate_config maybe?

Reply:
Just to confirm, you mean the message itself, right?

If so, do you prefer this form (providing the error class) or the cmd_validate_config form (sending the full trace)? I'm easy either way, though I think the former results in nicer notifications and I assume users of cmd_validate_config aren't using the notification window itself for thorough debugging.

Reply:
Yeah just the message. Personally I like to have the full trace directly so that I precisely know what I have to debug when it happens and do not need to look somewhere else, but I mostly want consistent stuff. :)

Reply:
Well your actual use case is higher priority than my hypothetical so I'll go that way ;)